app/views/pieces/show.html.erb

```diff
@@ -0,0 +1,22 @@
+<table class="chessboard">
+  <% 8.times do |row| %>
+    <tr>
+      <% 8.times do |column| %>
+        <% piece = Piece.where(x_cord: column, y_cord: row, game_id: @game).first %>
```

Comment:
Shouldn't we make this line the same as you did in games/show.html.erb?
`<% piece = @game.pieces.where(x_cord: column, y_cord: row).first %>`